packages/webapp/src/webview/state/reducers.ts

```diff
@@ -100,6 +113,10 @@ export const reducer: Reducer<AppState, GuidedAnswerActions> = (
             newState.betaFeatures = action.payload;
             break;
         }
+        case GUIDE_FEEDBACK: {
```

Comment:
See comment for `packages/webapp/src/webview/ui/components/GuidedAnswerNavPath/GuidedAnswerNavPath.tsx`